qa/rpc-tests/p2p-versionbits-warning.py

```diff
@@ -73,7 +76,10 @@ def setup_network(self):
         self.nodes.append(start_node(0, self.options.tmpdir, self.node_options))
 
         import re
-        self.vb_pattern = re.compile("^Warning.*versionbit")
+        # bip135 begin
+        # with new unknown version algo, warning has changed.
+        self.vb_pattern = re.compile("^Warning: Unknown block versions being mined Its possible unknown rules are in effect")
```

Comment:
Thanks, this does seem to be an error.

P.S. I will fix this, but also merge corrections I made to this test while porting the whole thing to Core. There were other changes which made the test clearer, such as that the actual error threshold is 51/100 (in this test, 1 is added to some thresholds, which is unnecessarily complicating things).